### PR TITLE
Init

### PR DESCRIPTION
**Pull request by iruzevic**
## Summary by CodeRabbit

- **New Features**
	- Introduced a Contributor Covenant Code of Conduct to foster an inclusive community.
	- Added a continuous integration workflow for automated linting and code standard checks.
	- Implemented a workflow for bundling assets into a release artifact.
	- Created a template for pull requests to standardize submissions.
	- Launched a plugin addon boilerplate for the Eightshift Forms plugin, including activation and deactivation hooks, and settings configuration.
- **Documentation**
	- Added a README file with essential information and quick start instructions for the Eightshift Forms Plugin - Addon Boilerplate.
	- Documented notable changes in the `CHANGELOG.md`.
- **Chores**
	- Established `.gitignore` and `.gitattributes` for better project hygiene.
	- Provided scripts for plugin bundling.
- **Refactor**
	- Defined custom filters, actions, and settings handling in the plugin's architecture.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+[![license](https://img.shields.io/github/license/infinum/eightshift-forms-addon-boilerplate.svg?style=for-the-badge)](https://github.com/infinum/eightshift-forms-addon-boilerplate)
+[![codecov](https://img.shields.io/codecov/c/gh/infinum/eightshift-forms-addon-boilerplate/branch/develop?label=Codecov&style=for-the-badge&token=02Lfa2jjoK)](https://codecov.io/gh/infinum/eightshift-forms-addon-boilerplate)
+
+# Eightshift Forms Plugin - Addon Boilerplate
+
+Addon Boilerplate is a starting point for creating new addons for the Eightshift Forms plugin.
+
+# Eightshift Forms Plugin
+
+Eightshift Forms plugins is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```
Comment (piqusy) [inline README.md]:
```suggestion
Eightshift Forms plugin is a complete form builder tool that utilizes modern Block editor features with multiple third-party integrations to boost your project to another level.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+If you have any questions or problems, please [open an issue](https://github.com/infinum/eightshift-forms/issues) on GitHub. 
+
+When submitting issues or otherwise participating in development, please follow our [code of conduct](https://github.com/infinum/eightshift-forms/blob/develop/CODE_OF_CONDUCT.md), and fill out the issue template properly. We'll do our best to answer your issues as quickly as humanly possible: following these steps helps us out a lot in doing that.
+
+## 👩‍💻 Maintainers 🧑‍💻 
+Eightshift Forms is maintained and sponsored by [Infinum](https://infinum.com).
+
+## ⚖️ License
+Eightshift Forms &copy;2024 [Infinum](https://infinum.com). It is payed software, and may be redistributed under the terms specified in the LICENSE file.
```
Comment (piqusy) [inline README.md]:
```suggestion
Eightshift Forms &copy;2024 [Infinum](https://infinum.com). It is paid software, and may be redistributed under the terms specified in the LICENSE file.
```
Is it paid tho'?

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice 👏🏼 

Left a few wording sugestions

File: src/Hooks/Filters.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+{
+	/**
+	 * Get list of all public filters.
+	 *
+	 * @return array<mixed>
+	 */
+	public static function getPublicFilters(): array
+	{
+		return [
+		];
```
Comment (ghost) [inline src/Hooks/Filters.php]:
```suggestion
		return [];
```

**Review by ghost — CHANGES_REQUESTED** [review]
👏

**Review by ghost — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Pull request merged**